### PR TITLE
User-Friendlier Gradle

### PR DESCRIPTION
**Pull request by wolfieboy09**
I first updated the felis dam pluging to `1.8.2-alpha`, and made upgrading dependencies into `gradle.properties`.
Mod version and MC version are properties as well

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1,9 @@
-org.gradle.jvmargs=-Xmx3G
\ No newline at end of file
+org.gradle.jvmargs=-Xmx3G
+
+mc_version=1.20.5
```
Comment (0xJoeMama) [inline gradle.properties]: can we expand to minecraft?

Comment (0xJoeMama) [reply gradle.properties]: Also 1.20.6 while you are at it...

File: gradle.properties [hunk 2]
```diff
@@ -1 +1,9 @@
...
+felis_version=1.3.0-alpha
```
Comment (0xJoeMama) [inline gradle.properties]: get the latest felis and kittens version from here: https://repsy.io/mvn/0xjoemama/public/felis/

**Review by 0xJoeMama — CHANGES_REQUESTED** [review]

**Comment by wolfieboy09** [comment]
Let me get the latest

**Review by wolfieboy09 — COMMENTED** [review]
a

**Review by wolfieboy09 — COMMENTED** [review]
a

**Review by 0xJoeMama — APPROVED** [review]

**Pull request merged**